sympy/physics/vector/tests/test_frame.py

```diff
@@ -451,10 +451,10 @@ def test_dcm_diff_16824():
     assert simplify(AwB.dot(A.y) - alpha2) == 0
     assert simplify(AwB.dot(B.y) - beta2) == 0
 
-def test_orient_explicit():
```

Comment:
Don't change the `orient_explicit` test because we still want to make sure it works. Instead, add a new `test_orient_dcm`, where you showcase that [this](https://github.com/sympy/sympy/issues/24764#issue-1596398197) works.